flow/raster_cache.cc

```diff
@@ -73,8 +93,12 @@ std::unique_ptr<RasterCacheResult> RasterCache::Rasterize(
     return nullptr;
   }
 
-  DlSkCanvasAdapter canvas(surface->getCanvas());
-  canvas.Clear(DlColor::kTransparent());
+  // Clear the surface canvas directly to avoid RTree storing a giant clear
+  // rect.
+  auto surfaceCanvas = surface->getCanvas();
+  surfaceCanvas->clear(SK_ColorTRANSPARENT);
+
+  DisplayListBuilder canvas(true);
```

Comment:
Could the RTree just be passed in from the caller? It should only be needed with DL cache items and they should already have an RTree in their DL.

Reply:
Good point. It looks like the `DisplayList` of `DisplayListLayer` is always created with an `RTree`. We shouldn't duplicate this process unnecessarily.